templates/server/src/main/resources/archetype-resources/src/main/java/__packageInPathFormat__/general/common/beanmapping/BeanMapper.java

```diff
@@ -0,0 +1,93 @@
+package ${package}.general.common.beanmapping;
+
+import java.util.List;
+import java.util.Set;
+
+/**
+ * This is the interface used to convert from one Java bean to another compatible bean (e.g. from a JPA entity to a
+ * corresponding transfer-object).
+ *
+ */
+public interface BeanMapper {
```

Comment:
This is something to be discussed. But lets arrange a meeting for this because it is rather complex and clarifying via reivew chat is not the best approach.

Reply:
I do not see why we are blocking progress this way. IMHO we can simply merge this and you are more than welcome to create PRs for improvement. I am happy to get orika out as well as it seems rather unmaintained and already has issues with Java > 17.